buildenv/jenkins/common/test

```diff
@@ -36,7 +36,6 @@ JAVA_BIN = "${WORKSPACE}/${BUILD_RELEASE_IMAGES}${JDK_FOLDER}${JRE_FOLDER}/bin"
 def fetch_artifacts(){
     stage('Fetch Artifacts') {
         timestamps {
-            cleanWs()
```

Comment:
This `cleanWs()` would be cleaning up the OpenJ9 repo from the `checkout scm` step. But it doesn't appear to be getting in the way right now if we leave it on disk during the test build
```
jenkins@LXEOJ901:~/jenkins-agent/workspace/Adam_Test-Sanity-JDK8-linux_390-64_cmprssptrs$ ls -lrt        
total 95068
drwxrwxr-x  9 jenkins jenkins     4096 Apr 27 00:36 j2sdk-image
-rw-rw-r--  1 jenkins jenkins  1896676 Apr 27 00:39 test-2d34dcf.tar.gz
-rw-rw-r--  1 jenkins jenkins 95277256 Apr 27 00:40 OpenJ9-JDK8-linux_390-64_cmprssptrs-201827040040.tar.gz
drwxrwxr-x  2 jenkins jenkins     4096 Apr 27 14:23 buildspecs
drwxrwxr-x  5 jenkins jenkins     4096 Apr 27 14:23 buildenv
-rw-rw-r--  1 jenkins jenkins    11357 Apr 27 14:23 asl-v20.text
drwxrwxr-x  2 jenkins jenkins     4096 Apr 27 14:23 artwork
-rw-rw-r--  1 jenkins jenkins     2197 Apr 27 14:23 about.html
-rw-rw-r--  1 jenkins jenkins     5054 Apr 27 14:23 README.md
-rw-rw-r--  1 jenkins jenkins    33883 Apr 27 14:23 LICENSE
-rw-rw-r--  1 jenkins jenkins     9644 Apr 27 14:23 CONTRIBUTING.md
drwxrwxr-x  5 jenkins jenkins     4096 Apr 27 14:23 debugtools
drwxrwxr-x  4 jenkins jenkins     4096 Apr 27 14:23 jcl
-rw-rw-r--  1 jenkins jenkins    17300 Apr 27 14:23 epl-2.0.html
drwxrwxr-x  6 jenkins jenkins     4096 Apr 27 14:23 doc
-rw-rw-r--  1 jenkins jenkins     9183 Apr 27 14:23 notice.html
-rw-rw-r--  1 jenkins jenkins    16149 Apr 27 14:23 longabout.html
drwxrwxr-x 94 jenkins jenkins     4096 Apr 27 14:23 runtime
drwxrwxr-x 15 jenkins jenkins     4096 Apr 27 14:23 sourcetools
drwxrwxr-x  6 jenkins jenkins     4096 Apr 27 14:23 test
drwxrwxr-x  4 jenkins jenkins     4096 Apr 27 14:26 openj9
```
You can see the OpenJ9 repo as well as the 2 tar artifacts, and the uncompressed version of these `openj9` and `j2sdk-image`
https://ci.eclipse.org/openj9/view/Sandbox/job/Adam_Test-Sanity-JDK8-linux_390-64_cmprssptrs/11/console
It doesn't matter too much because this code will be getting replaced shortly